### PR TITLE
Add introduction slide for Basic HTML and HTML5 section

### LINKED ISSUE #12624: Introduction missing for Basic HTML and HTML5 section
#### Section Name

The Beta site Basic HTML and HTML5 section

#### Issue Description

This section currently lacks a landing page with introductory text.

### PR DESCRIPTION
**Pull request by HKuz**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #12624 

#### Description

PR adds the introduction text for the Basic HTML and HTML5 section.

### CODE REVIEW

File: seed/challenges/01-responsive-web-design/basic-html-and-html5.json [hunk 1]
```diff
@@ -5,6 +5,36 @@
   "helpRoom": "Help",
   "challenges": [
     {
+      "id": "587d774b367417c4c2512a9b",
+      "title": "Introduction to the Basic HTML and HTML5 Challenges",
+      "description": [
+        [
+          "",
+          "",
+          "HTML, or HyperText Markup Language, is a markup language to describe the structure of a web page. It uses a special syntax of elements to organize and give information about your content to the browser. Elements usually have opening and closing tags that surround and give meaning to content. For example, there are different tag options to place around text to show whether it's a heading, a paragraph, or a list.<br><br>The HyperText part of HTML comes from the early days of the web. Pages usually contained static documents that connected to other documents with hypertext links. As web pages and web applications grow more complex, the W3 Consortium updates HTML. The latest version, or specification, is HTML5.<br><br>This section introduces how to use HTML elements to give structure and meaning to your web content.",
```
Comment (raisedadead) [inline seed/challenges/01-responsive-web-design/basic-html-and-html5.json]: Can we have this over multiple slides, this one looks too long for a read?

Comment (HKuz) [inline seed/challenges/01-responsive-web-design/basic-html-and-html5.json]: Will do - I'll move the 2nd and 3rd paragraphs (the 3rd is only 1 sentence long) onto a second slide.

Comment (raisedadead) [inline seed/challenges/01-responsive-web-design/basic-html-and-html5.json]: Thanks @HKuz !

**Review by raisedadead — APPROVED** [review]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @HKuz

**Comment by HKuz** [comment]
Awesome, thanks @raisedadead 💯!

**Pull request merged**